cms/templates/settings.html

```diff
@@ -248,13 +251,61 @@ <h2 class="title-2">${_('Course Schedule')}</h2>
+                  <select id="certificates-display-behavior">
+                    <option value="early_no_info">${_("Immediately upon passing")}</option>
+                    <option value="end">${_("End date of course")}</option>
+                    <option value="end_with_date">${_("A date after the course end date")}</option>
+                  </select>
+                % else:
+                  <input id="certificates-display-behavior" type="text">
+                % endif
+                <span class="tip tip-stacked">${_("Certificates are awarded at the end of a course run")}</span>
+
+                % if use_v2_cert_display_settings:
+                  <!-- Collapsible -->
+                  <div class="collapsible">
+                    <div id="certificate-display-behavior-collapsible-trigger" class="collapsible-trigger" role="button" tabindex="0" aria-expanded="false">
+                      <span>
+                        <span class="icon icon-inline fa fa-info-circle" aria-hidden="true"></span>
+                        Read more about this setting
+                      </span>
+                    </div>
+                    <div id="certificate-display-behavior-collapsible-content" class="collapsible-content collapsed">
+                      <p>In all configurations of this setting, certificates are generated for learners as soon as they achieve the passing threshold in the course (which can occur before a final assignment based on course design)</p>
+                      <div>
+                        <div class="collapsible-description-heading">Immediately upon passing</div>
+                        <div class="collapsible-description-description">Learners can access their certificate as soon as they achieve a passing grade above the course grade threshold. Note: learners can achieve a passing grade before encountering all assignments in some course configurations.</div>
+                      </div>
+                      <div>
+                        <div class="collapsible-description-heading">On course end date</div>
+                        <div class="collapsible-description-description">Learners with passing grades can access their certificate once the end date of the course has elapsed.</div>
+                      </div>
+                      <div>
+                        <div class="collapsible-description-heading">A date after the course end date</div>
+                        <div class="collapsible-description-description">Learners with passing grades can access their certificate after the date that you set has elapsed.</div>
+                      </div>
+                    </div>
+                  </div>
+                % endif
+              </div>
+
+              % if use_v2_cert_display_settings:
+                <div class="field date hidden" id="field-certificate-available-date" >
+              % else:
+                <div class="field date" id="field-certificate-available-date" >
```

Comment:
This is the _old_ settings, right? So if we're using v2, we're hiding the older sections of the advanced settings page?

(This is my first time looking at _anything_ in Studio)

Reply:
Ah! You just reminded of a call out I meant to add to the main description! Copied from the updated description: 
> This code is all toggleable except for the transition of Certificate Display Behavior to the main settings page (instead of the advanced settings page). Because of the code paths and the way this data is stored, I couldn't have it easily switch between the two places.

When the feature is off, it just shows a normal text field instead of a drop down that hides CAD.